db/migrate/20161025201812_add_user_to_references.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToReferences < ActiveRecord::Migration[5.0]
+  def change
+    add_reference :references, :user, foreign_key: true
```

Comment:
This is generated by `rails generate migration AddUserToReferences user:references`

And yeah, you could just do an integer, too. I'm not sure what the `foreign_key: true` bit does